lib/engine/game/trainless_shares_half_value.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+#
+# This module makes the shares of companies without a train worth half the usual value
```

Comment:
The player value here also assumes a player can sell a presidents share to the market - which is somewhat orthogonal to trainless companies having price shares. So far both have been true together, but that might not necessarily be always true.

Reply:
Great catch, thanks Steve - I will get this to look for the flag on the game and react accordingly.

Reply:
The original liquidity calculation is already using `bundles_for_corporation`, so this turned out to actually be quite a simplification - just needed to account for `PRESIDENT_SALES_TO_MARKET` in one more place. Testing now and will add a commit.